npm/css-src/sass/_header.scss

```diff
@@ -15,7 +15,10 @@ $govuk-header-link: govuk-colour('white');
 .govuk-header__tna-content {
   margin: 0 auto;
   position: relative;
-  width: 290px;
+}
+
+.tna-header {
+  display: flex;
```

Comment:
I'll try to have a look at this whole PR later today, but have you seen how other services like [Notify](https://www.notifications.service.gov.uk/) and [Pay](https://www.payments.service.gov.uk/) add links to the right-hand-side of the navigation using the design system? It looks like they're using `.govuk-header__navigation-item`.

Reply:
Yeah, I did try it but because the other elements in that header are using our own custom classes `govuk-header__tna-content` and `govuk-header__tna-logo` so I couldn't get the navigation-item element looking right. I think it's fine if everything else is design system standard but here, they're not.

Reply:
I've added a comment to the card for adding the user's name so that we revisit all the CSS for the header: https://national-archives.atlassian.net/browse/TDR-796?focusedCommentId=14620

I think there's probably a simpler way to do this, but it probably makes sense to wait until we're adding the final element.

Reply:
Ok cool. Flexbox is pretty good. I did try to put the user name element in as well to see what it looked like and it was pretty easy to add with this css but as you say, we can wait until we need to do it.